x/icqoracle/types/keys.go

```diff
@@ -0,0 +1,26 @@
+package types
+
+import fmt "fmt"
+
+const (
+	ModuleName = "icqoracle"
+
+	// StoreKey defines the primary module store key
+	StoreKey = ModuleName
+
+	// RouterKey defines the routing key
+	RouterKey = ModuleName
+)
+
+var (
+	ParamsKey        = []byte("params")
+	PriceQueryPrefix = []byte("pricequery")
+)
+
+func TokenPriceQueryKey(baseDenom, quoteDenom, poolId string) []byte {
+	return []byte(fmt.Sprintf("%s|%s|%s", baseDenom, quoteDenom, poolId))
+}
+
+func TokenPriceByDenomKey(baseDenom string) []byte {
+	return []byte(baseDenom)
+}
```

Comment:
I think you can remove this and just do the `[]byte(TokenPrice.BaseDenom)` in the keepers